_bookdown.yml

```diff
@@ -7,6 +7,7 @@ rmd_files: ["index.Rmd",
             "02-why-R.Rmd",
             "03-components.Rmd",
             "04-rstudio-tour.Rmd",
+            "04.5-rmarkdown.Rmd",
```

Comment:
We should probably just fix the numbering but this is fine for now.

Reply:
yes I plan to- but didn't want to mess up other PRs